<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playlist::UI::ScannerView</class>
 <widget class="QWidget" name="ScannerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>25</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QProgressBar" name="scanProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="itemsProgress">
     <property name="text">
      <string notr="true">0/0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="scanPause">
     <property name="icon">
      <iconset>
       <normalon>:/playback/pause.png</normalon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Pause/continue scanning</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="scanCancel">
     <property name="icon">
      <iconset>
       <normalon>:/playback/stop.png</normalon>
      </iconset>
     </property>
     <property name="toolTip">
      <string>Stop scanning</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
